GitExtensions/Program.cs

```diff
@@ -204,8 +204,7 @@ private static void HandleConfigurationException(System.Configuration.Configurat
                                 if (DialogResult.OK.Equals(MessageBox.Show(string.Format("Files have been deleted.{0}{0}Would you like to attempt to restart GitExtensions?", Environment.NewLine), "Configuration Error", MessageBoxButtons.OKCancel, MessageBoxIcon.Information)))
                                 {
                                     var args = Environment.GetCommandLineArgs();
-                                    System.Diagnostics.Process p = new System.Diagnostics.Process();
-                                    p.StartInfo.FileName = args[0];
+                                    var p = new System.Diagnostics.Process { StartInfo = { FileName = args[0] } };
```

Comment:
💭 I don't see how this is an improvement

Reply:
Again a matter of taste. I like construction as an expression where possible, and hope event subscriptions can be added to object initialisers one day. I've seen some discussion about this on csharplang.